nixos/tests/yabar.nix

```diff
@@ -1,4 +1,4 @@
-import ./make-test.nix ({ pkgs, lib }:
+import ./make-test.nix ({ pkgs, lib, system }:
```

Comment:
could you add  `system ? builtins.currentSystem` here? Many tests do this, it makes them easier to run standalone (without `release.nix`).